From f5e8736a084945bc83194845c7d06cc57c1d7cd5 Mon Sep 17 00:00:00 2001
From: Yang Fan F <fan.f.yang@intel.com>
Date: Wed, 29 Apr 2015 01:11:31 +0800
Subject: [PATCH 03/28] Revert "camera ov5693 and mt9m114: device power_up
 retries."

This reverts commit 56a6c78ad1845c99e4bd526287f9b876bd9f1704.

Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9279
Change-Id: I5df2688d90db9c463a04e9b6a86f5e00549fb86f
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/440489
---
 .../camera/drivers/media/i2c/mt9m114.c                 | 17 +----------------
 .../camera/drivers/media/i2c/ov5693-ecs/ov5693.c       | 18 ++----------------
 2 files changed, 3 insertions(+), 32 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
index 53ecbbf..a1b2f36 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/mt9m114.c
@@ -536,7 +536,7 @@ static int gpio_ctrl(struct v4l2_subdev *sd, bool flag)
 	return ret;
 }
 
-static int __power_up(struct v4l2_subdev *sd)
+static int power_up(struct v4l2_subdev *sd)
 {
 	struct mt9m114_device *dev = to_mt9m114_sensor(sd);
 	struct i2c_client *client = v4l2_get_subdevdata(sd);
@@ -609,21 +609,6 @@ static int power_down(struct v4l2_subdev *sd)
 	return ret;
 }
 
-static int power_up(struct v4l2_subdev *sd)
-{
-	static const int retry_count = 4;
-	int i, ret;
-
-	for (i = 0; i < retry_count; i++) {
-		ret = __power_up(sd);
-		if (!ret)
-			return 0;
-
-		power_down(sd);
-	}
-	return ret;
-}
-
 static int mt9m114_s_power(struct v4l2_subdev *sd, int power)
 {
 	if (power == 0)
diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.c b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.c
index 2135828..0c93e9f 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.c
@@ -1384,7 +1384,8 @@ static int gpio_ctrl(struct v4l2_subdev *sd, bool flag)
 	return ret;
 }
 
-static int __power_up(struct v4l2_subdev *sd)
+
+static int power_up(struct v4l2_subdev *sd)
 {
 	struct ov5693_device *dev = to_ov5693_sensor(sd);
 	struct i2c_client *client = v4l2_get_subdevdata(sd);
@@ -1464,21 +1465,6 @@ static int power_down(struct v4l2_subdev *sd)
 	return ret;
 }
 
-static int power_up(struct v4l2_subdev *sd)
-{
-	static const int retry_count = 4;
-	int i, ret;
-
-	for (i = 0; i < retry_count; i++) {
-		ret = __power_up(sd);
-		if (!ret)
-			return 0;
-
-		power_down(sd);
-	}
-	return ret;
-}
-
 static int ov5693_s_power(struct v4l2_subdev *sd, int on)
 {
 	int ret;
-- 
1.9.1

